### PR TITLE
fix: Update loading copy for Blockaid on mobile

### PR DESCRIPTION
**Pull request by blackdevelopa**
## **Description**

This PR updates the copy displayed when loading blockaid response

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1664

## **Manual testing steps**

1. Enable blockaid settings from Settings > Experimental
2. Attempt to trigger any transaction. Can use the [test dapp](https://metamask.github.io/test-dapp/)

### **Before**

Screenshot 2023-11-13 at 10 39 51

Screenshot 2023-11-13 at 10 40 08

### **After**

https://github.com/MetaMask/metamask-mobile/assets/29962968/67f1e53e-b6e2-4330-a7d7-2f2cc0d9d491

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/components/UI/BlockaidBanner/BlockaidBanner.tsx [hunk 1]
```diff
@@ -122,11 +122,8 @@ const BlockaidBanner = (bannerProps: BlockaidBannerProps) => {
       return (
         <View style={styles.bannerWrapperMargined}>
           <BannerAlert
-            severity={BannerAlertSeverity.Info}
-            title={strings('blockaid_banner.no_risks')}
-            onClose={() => {
-              setDisplayPositiveResponse(false);
-            }}
+            severity={BannerAlertSeverity.Warning}
+            title={strings('blockaid_banner.warning_desc')}
```
Comment (jpuri) [inline app/components/UI/BlockaidBanner/BlockaidBanner.tsx:126]: @blackdevelopa :  this change should go in line 106.

Comment (jpuri) [reply app/components/UI/BlockaidBanner/BlockaidBanner.tsx:126]: This banner is to be displayed after successful validation.

Comment (blackdevelopa) [reply app/components/UI/BlockaidBanner/BlockaidBanner.tsx]: Updated here. https://github.com/MetaMask/metamask-mobile/pull/7799/commits/2709e55b9226a543b5c4d9e0c3ea15c32a069e2f TY

File: app/components/UI/BlockaidBanner/BlockaidBanner.tsx [hunk 2]
```diff
@@ -123,10 +123,7 @@ const BlockaidBanner = (bannerProps: BlockaidBannerProps) => {
         <View style={styles.bannerWrapperMargined}>
           <BannerAlert
             severity={BannerAlertSeverity.Info}
-            title={strings('blockaid_banner.no_risks')}
-            onClose={() => {
-              setDisplayPositiveResponse(false);
-            }}
+            title={strings('blockaid_banner.loading_complete_title')}
```
Comment (jpuri) [inline app/components/UI/BlockaidBanner/BlockaidBanner.tsx:126]: Hey on close above is required I think.

Comment (blackdevelopa) [reply app/components/UI/BlockaidBanner/BlockaidBanner.tsx:126]: Updated. TY

**Review by segun — APPROVED** [review]

**Review by jpuri — APPROVED** [review]

**Pull request merged**